src/test/daml/Daml/Finance/Account/Test/Util.daml

```diff
@@ -0,0 +1,25 @@
+module Daml.Finance.Account.Test.Util where
+
+import DA.Set qualified as S (empty, fromList, singleton)
+import Daml.Finance.Interface.Account.Account qualified as Account (Controllers(..))
+
+-- | Options for transfer controllers.
+data ControlledBy
+  = Owners
+    -- ^ Owners-controller.
+  | Custodian
+    -- ^ Custodian-controller.
```

Comment:
```suggestion
    -- ^ Custodian controls inbound and outbound transfers
```

Alternatively, incoming and outgoing transfers? Transfers from the account and to the account?